.github/CONTRIBUTING.md

```diff
@@ -79,7 +79,7 @@ To run the link-checker, follow these steps:
 
 ### Code Editor
 
-You should use the multi-platform [Visual Studio Code (VS Code)][use-vscode-editor].
+You should use the multi-platform [Visual Studio Code (VSCode)][use-vscode-editor].
```

Comment:
Why do we want to change this? `VS Code` is used on VS Code home site.

Reply:
I'm pretty sure it is one word

Reply:
Everywhere I see it, it's `VS Code` or `Visual Studio Code` 🤷‍♂️

Reply:
reverted